packages/findutils.rb

```diff
@@ -24,18 +24,6 @@ class Findutils < Package
 
   depends_on 'python3'
 
-  ENV['PYTHON'] = 'python3' # Force use of python3 over python2.7
```

Comment:
Should this be set in `#{CREW_PREFIX}/etc/env.d/00-python3`?  Probably should be done in the python3 package anyway.

Reply:
Created https://github.com/chromebrew/chromebrew/issues/8433 to handle this with the next Python 3.x update we put out.